### DISCUSSION TITLE
increment & decrement won't even work

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#2159] [Q&A] [answered]
If a number isn't solely present then the increment & decrement operators don't even work.
```css
1px 2 px
```
I was just editing a CSS file and first of all I have to place my cursor over the number and then use the increment or decrement operation. And if the word is alphanumeric then I have to place spaces around the number then place my cursor back on the number and now the increment and decrement operations work. I'm not sure if that's how it's intended to work but my complain about increment and decrement operation was quite old. In almost 90% of the cases I have a non-numeric character along with the number I'd like to increment.

Is this supposed to be fixed? I've tried it in helix from Arch as well as the one I built from source.

**Comment by TornaxO7** [comment, marked as answer]
Hi! According to the tutor (`:h tutor`):
```
 Press Ctrl-a to increment the number under selection.
 Press Ctrl-x to decrement the number under selection.
```
so I assume the answer is: Yes, that' intended.
